org.eclipse.buildship.core/src/main/java/org/eclipse/buildship/core/GradleBuild.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright (c) 2018 the original author or authors.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+
+package org.eclipse.buildship.core;
+
+import org.eclipse.core.runtime.IProgressMonitor;
+
+/**
+ * Represents a Gradle build that can be imported into or synchronized with the workspace.
```

Comment:
...and many other things. I don't think this adds much to the description, since I can see what I can do with this by looking at the methods. If there is nothing more to say than "Represents a Gradle build", that's fine.

Reply:
I've added some documentation to the interace.

Reply:
Makes sense. I've removed the unnecessary parts.